TODO.md

```diff
@@ -0,0 +1,9 @@
+# Do not merge this pull request until the following is done:
+[x] Make sure rebuild actually works.
+[.] Separate rtop repo/package.
+   [x] created repo:  https://github.com/reasonml/rtop
+   [ ] Move rtop portion there.
+[x] Compare old .install file to new one to make sure we're not missing anything.
```

Comment:
I'm surprised to see this comment. I thought people at Facebook understood the value mono repos bring.

Let me just note that jbuilder easily supports multiple packages out of 1 repo. So you're only getting a second git history and no extra modularity from such a split.

Reply:
Side note: I don't really think "monorepo" is a meaningful term with the way it's often used, but for any definition, having multiple packages under development in one repo breaks down when it comes time to consumer or produce code for people outside of that organization. But that aside - for my particular grievance here, it doesn't matter if we do use a "monorepo", because we still need to push these individual packages to opam and that particular process is the painful part - not creating multiple packages, or using them in a "monorepo" - but submitting the physical packages to opam when they depend on each other. It takes one day to submit an opam-repository request, and I can't submit the one that depends on the first until the first is merged otherwise the second package's build will fail. The reverse matrix is super cool don't get me wrong, but it typically ends up taking *two* entire days to release two packages. I understand the value of having a stable opam universe, and so I'm okay with that - but we need tools that let us rapidly release our own stack, (and get others to instantly consume that stack) bypassing opam-repository, and then gradually sync those multiple released packages to opam.

Reply:
>  It takes one day to submit an opam-repository request, and I can't submit the one that depends on the first until the first is merged otherwise the second package's build will fail

Just submit them both in the same PR, and they will be batch tested individually with the right dependency checks.

Reply:
@avsm Thanks, we're investigating a couple of approaches to doing that. It would be great to be able to keep multiple opam packages in one repo.

"and they will be batch tested individually with the right dependency checks."

Do you mean they will be batch tested simultaneously?